server/src/test/java/org/apache/gravitino/server/web/rest/TestOwnerOperations.java

```diff
@@ -0,0 +1,221 @@
+import org.apache.gravitino.dto.authorization.OwnerDTO;
+import org.apache.gravitino.dto.requests.OwnerSetRequest;
+import org.apache.gravitino.dto.responses.ErrorConstants;
+import org.apache.gravitino.dto.responses.ErrorResponse;
+import org.apache.gravitino.dto.responses.MessageResponse;
+import org.apache.gravitino.dto.responses.OwnerResponse;
+import org.apache.gravitino.dto.responses.SetResponse;
+import org.apache.gravitino.exceptions.NotFoundException;
+import org.apache.gravitino.lock.LockManager;
+import org.apache.gravitino.rest.RESTUtils;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.glassfish.jersey.test.TestProperties;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class TestOwnerOperations extends JerseyTest {
+  private static final OwnerManager manager = mock(OwnerManager.class);
+
+  private static class MockServletRequestFactory extends ServletRequestFactoryBase {
+    @Override
+    public HttpServletRequest get() {
+      HttpServletRequest request = mock(HttpServletRequest.class);
+      when(request.getRemoteUser()).thenReturn(null);
+      return request;
+    }
+  }
+
+  @BeforeAll
+  public static void setup() throws IllegalAccessException {
+    Config config = mock(Config.class);
+    Mockito.doReturn(100000L).when(config).get(TREE_LOCK_MAX_NODE_IN_MEMORY);
+    Mockito.doReturn(1000L).when(config).get(TREE_LOCK_MIN_NODE_IN_MEMORY);
+    Mockito.doReturn(36000L).when(config).get(TREE_LOCK_CLEAN_INTERVAL);
+    FieldUtils.writeField(GravitinoEnv.getInstance(), "lockManager", new LockManager(config), true);
+    FieldUtils.writeField(GravitinoEnv.getInstance(), "ownerManager", manager, true);
+  }
+
+  @Override
+  protected Application configure() {
+    try {
+      forceSet(
+          TestProperties.CONTAINER_PORT, String.valueOf(RESTUtils.findAvailablePort(2000, 3000)));
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+
+    ResourceConfig resourceConfig = new ResourceConfig();
+    resourceConfig.register(OwnerOperations.class);
+    resourceConfig.register(
+        new AbstractBinder() {
+          @Override
+          protected void configure() {
+            bindFactory(MockServletRequestFactory.class).to(HttpServletRequest.class);
+          }
+        });
+
+    return resourceConfig;
+  }
+
+  @Test
+  void testGetOwnerForObject() {
+    Owner owner =
+        new Owner() {
+          @Override
+          public String name() {
+            return "test";
+          }
+
+          @Override
+          public Type type() {
+            return Type.USER;
+          }
+        };
+
+    when(manager.getOwner(any(), any())).thenReturn(Optional.of(owner));
+
+    Response resp =
+        target("/metalakes/metalake1/owners/metalake/metalake1")
```

Comment:
I think this request path is very confusing.
If we need process schema, then the request path is `/metalakes/metalake1/owners/metalake/metalake1/catalog1/schema1` ?
Cloud there be a better design?
`/metalake1/catalog1/schema1/owner` ?

Reply:
/metalakes/metalake1/owners/schema/metalake1.catalog1.schema1. This style is the same as the tag's.

Reply:
OK